tests/k6/tests/serviceowner/authentication.js

```diff
@@ -45,56 +45,46 @@ export default function () {
             let r = getSO("dialogs/" + uuidv4(), params);
             expectation(r);
         });
-    
+
```

Comment:
Intentional?

Reply:
Removing whitespace? Yes/no, happens automatically in Rider :bowtie:

Reply:
Don't see these whitespace-only changes in GitHub either, 
https://addons.mozilla.org/en-US/firefox/addon/github-whitespace-disabler/ 
if ya want 😊 (adds ?w=1 to the URL)